pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -871,8 +872,8 @@ public void ensureAllPartitionsConsuming(TableConfig tableConfig, StreamConfig s
       boolean offsetsHaveToChange = offsetCriteria != null;
       if (isTableEnabled && !isTablePaused) {
         List<PartitionGroupConsumptionStatus> currentPartitionGroupConsumptionStatusList =
-            offsetsHaveToChange
-                ? Collections.emptyList() // offsets from metadata are not valid anymore; fetch for all partitions
+            offsetsHaveToChange ? Collections.emptyList()
```

Comment:
(minor) This is not as readable. Suggest not reformatting unrelated code

Reply:
Reverted this change.